lensterxyz/lenster
# feat: add message button to profile using xmtp

## What does this PR do?

This PR adds a message button placeholder to Profile behind the `messages` feature flag to act as a prototype of sending a `gm` message via XMTP.

Note: currently the button is not styled to spec yet. Also, we are not yet handling the case if the person on the receiving end of the message isn't on XMTP. Those will be added as follow-ups!

<details>
<summary>Screenshots</summary>

| With `messages` FF | Without `messages` FF |
|--|--|
| <img width="439" alt="Screen Shot 2022-10-03 at 3 25 20 PM" src="https://user-images.githubusercontent.com/556051/193668548-b36f00e1-3388-4667-affd-7f128ede46c0.png">  |  <img width="437" alt="Screen Shot 2022-10-03 at 3 24 58 PM" src="https://user-images.githubusercontent.com/556051/193668587-c556a8c8-49c9-4d26-9ff4-daef6ff02e01.png">|
</details>
## How should this be tested?



- Sign in on testnet (or allowlist your profile in the `messages` feature flag)
- Navigate to someone else's profile (elisetest.test is on the XMTP network)
- Tap message
- You should be prompted to sign in with XMTP
- A `gm` message should be sent to the receiving profile as long as they are on XMTP
